### DISCUSSION TITLE
`Unmanaged Memory` leek?

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#10577] [Q&A] [answered]
Open the window and close many time, the `Unmanaged Memory` use will increase.
![2M8D T4Z Z B )DYZYL}@KH](https://user-images.githubusercontent.com/26276037/223367753-9c726fda-6937-48a6-8633-16364c535ad7.png)
![}F6GS}1Q1 74~}H}E`%{U0K](https://user-images.githubusercontent.com/26276037/223367764-aae88513-ad1c-43a2-ae38-99bc1ba868fb.png)

**Comment by Coloryr** [comment]
![QYBLEC5 SMTY $5`(M}4TES](https://user-images.githubusercontent.com/26276037/223368197-9ab8a23b-8869-47eb-8b28-149b30381266.png)

**Comment by Coloryr** [comment]
![E6)RBG4GT1PQQV8{%R_7)0K](https://user-images.githubusercontent.com/26276037/223368659-8e726ad5-347c-4064-b659-79c71aac7e89.png)

**Comment by maxkatz6** [comment, marked as answer]
~250mb doesn't seem exceptionally big for the typical .NET application. Can you check if any window is kept in the memory? Or opening and closing hundreds of windows to see if it grows and stays around 1gb or more?
In general, .NET GC might not release all the memory back to the OS right away after it was used.

**Reply by Coloryr** [reply]
Only `Open` and `Close` not increase

**Reply by Coloryr** [reply]
https://user-images.githubusercontent.com/26276037/223374914-16dd9fce-44c3-46d3-8352-8842e48961dc.mp4

**Comment by Coloryr** [comment]
![L5(K`B77K`WNE(Z( %39~$C](https://user-images.githubusercontent.com/26276037/223375213-2049b844-dfdf-4834-adaa-97dc7528f67e.png)
